### DISCUSSION TITLE
Modify the locations of saving images

### DISCUSSION THREAD

**Discussion by greechalf** [lllyasviel/Fooocus#3471] [Q&A] [answered]
I cannot locate the codes to modify the paths of saving images.

**Comment by mashb1t** [comment, marked as answer]
You can set/adjust the path_outputs in your config.txt, see example in example config file.

**Reply by bsdesigner** [reply]
Sorry to be off topic, but wanted to ask you guys. Have there been any updates today? Yesterday everything was fine, but today Stability Matrix updated Fooocus and it started crashing.... It loads the model, starts generating an image and then switches off after the first generation:


https://github.com/user-attachments/assets/b73d1279-47ae-42df-b4f7-45e07940522c

The first image is not saved, and when I open the History Log I see this:

![1](https://github.com/user-attachments/assets/3e3b3dea-521a-41a0-9157-da0a9400c5e1)

**Reply by mashb1t** [reply]
@bsdesigner there hsve been 2 bugfixes published today, but only concerning the config, not image generation.
Ensure that you can write to the path you've changed the config to and also check your (terminal) logs.

**Reply by bsdesigner** [reply]
> @mashb1t By trial and error, I finally figured out what the problem was. And the matter was in my inattention ))) The thing is that in the default.json file I had 4 Lora that were to be loaded at startup and naturally Fooocus processed them. But I completely forgot that I deleted one of these Lora, and I forgot to inform Fooocus about it ))) So, by deleting this Lora from the default preset, I solved this problem caused by my inattention... I apologise for distracting you from your work with such a trifling problem!
